src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.scss

```diff
@@ -18,4 +18,12 @@ staff-task-list {
   top: 50%;
   left: 50%;
   transform: translateX(-50%) translateY(50%);
-}
\ No newline at end of file
+}
+
+status-icon {
+  width: 100%;
+  min-width: 40px;
+  min-height: 40px;
+  font-size: 1.25em;
+  display: inline-block;
+}
```

Comment:
Here

Reply:
That was my question really :) - how do we do this better?

Can we apply this within the component itself?

Reply:
The was originally in 'src/app/common/status-icon/status-icon.scss'